homeassistant/__main__.py

```diff
@@ -94,13 +45,25 @@ def ensure_config_path(config_dir):
     """ Gets the path to the configuration file.
         Creates one if it not exists. """
 
+    lib_dir = os.path.join(config_dir, 'lib')
+
     # Test if configuration directory exists
     if not os.path.isdir(config_dir):
-        print(('Fatal Error: Unable to find specified configuration '
-               'directory {} ').format(config_dir))
-        sys.exit()
-
-    import homeassistant.config as config_util
+        try:
+            os.mkdir(config_dir)
```

Comment:
We should only try to make it if we are using the default one. Imaging making a typo and Home Assistant is creating a second dir !